Kitodo/src/main/java/org/kitodo/production/forms/createprocess/SearchTab.java

```diff
@@ -0,0 +1,127 @@
+     */
+    public Process getTemplateProcess() {
+        return templateProcess;
+    }
+
+    /**
+     * Set templateProcess.
+     *
+     * @param templateProcess as java.lang.Process
+     */
+    public void setTemplateProcess(Process templateProcess) {
+        this.templateProcess = templateProcess;
+    }
+
+    /**
+     * Get Process templates.
+     *
+     * @return list of SelectItem objects
+     */
+    public List<SelectItem> getProcessesForChoiceList() {
+        return SelectItemList.getProcessesForChoiceList();
+    }
+
+    /**
+     * Auswahl des Prozesses auswerten.
+     */
+    public String evaluateTemplateSelection() {
+        readTemplateSelection();
+
+        try {
+            URI uri = ServiceManager.getProcessService().getMetadataFileUri(templateProcess);
+            this.templateProcessWorkpiece = ServiceManager.getMetsService().loadWorkpiece(uri);
+        } catch (IOException e) {
+            Helper.setErrorMessage(CreateProcessForm.ERROR_READING, new Object[] {"template-metadata" }, logger, e);
+        }
+        return null;
+    }
+
+    private void readTemplateSelection() {
+        readTemplateWorkpieces(this.createProcessForm.getAdditionalDetailsTab().getAdditionalDetailsTableRows(),
+                this.templateProcess);
+        readTemplateTemplates(this.createProcessForm.getAdditionalDetailsTab().getAdditionalDetailsTableRows(),
+                this.templateProcess);
+        readTemplateProperties(this.createProcessForm.getProcessDataTab().getDigitalCollections(),
+                this.templateProcess);
+    }
+
+    private void readTemplateWorkpieces(List<AdditionalDetailsTableRow> additionalFields, Process processForChoice) {
+        for (Property workpieceProperty : processForChoice.getWorkpieces()) {
+            for (AdditionalDetailsTableRow row : additionalFields) {
+                if (row.getLabel().equals(workpieceProperty.getTitle())) {
+                    this.createProcessForm.getAdditionalDetailsTab().setAdditionalDetailsRow(row,
+                            workpieceProperty.getValue());
+                }
+                if (workpieceProperty.getTitle().equals("DocType")) {
+                    this.createProcessForm.getProcessDataTab().setDocType(workpieceProperty.getValue());
+                }
+            }
+        }
+    }
+
+    private void readTemplateTemplates(List<AdditionalDetailsTableRow> additionalFields, Process processForChoice) {
+        for (Property templateProperty : processForChoice.getTemplates()) {
+            for (AdditionalDetailsTableRow row : additionalFields) {
+                if (row.getLabel().equals(templateProperty.getTitle())) {
+                    this.createProcessForm.getAdditionalDetailsTab().setAdditionalDetailsRow(row,
+                            templateProperty.getValue());
+                }
+            }
+        }
+    }
+
+    // TODO: check whether we only need digital collections as "metadata" in the future
```

Comment:
Answer: Digital Collection does not require special treatment, but should be included in metadata.